lib/v2/njxzc/maintainer.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
     '/tzgg': ['real-jiakai'],
+    '/libtzgg': ['real-jiakai'],
```

Comment:
```suggestion
    '/libtzgg': ['real-jiakai'],
    '/tzgg': ['real-jiakai'],
```
Please insert new entries in alphabetical order